src/Microsoft.DotNet.Interactive.Formatting/HtmlFormatter.cs

```diff
@@ -11,30 +11,20 @@ namespace Microsoft.DotNet.Interactive.Formatting
 {
     public static class HtmlFormatter
     {
-        static HtmlFormatter()
-        {
-            Formatter.Clearing += (sender, args) => DefaultFormatters = new DefaultHtmlFormatterSet();
-        }
-
-        public static ITypeFormatter Create(
-            Type type,
-            bool includeInternals = false)
-        {
-            var genericCreateForAllMembers = typeof(HtmlFormatter<>)
-                                             .MakeGenericType(type)
-                                             .GetMethod(nameof(HtmlFormatter<object>.Create), new[]
-                                             {
-                                                 typeof(bool)
-                                             });
-
-            return (ITypeFormatter) genericCreateForAllMembers.Invoke(null, new object[]
-            {
-                includeInternals
-            });
-        }
+        public static ITypeFormatter GetBestFormatterFor(Type type) =>
```

Comment:
Can we find a name for these methods that's clearer about the intent? What makes a formatter "best"?

Reply:
Hmmm.... Maybe `GetCurrentFormatterFor` or `GetFormatterFor` or `GetPreferredFormatterFor` (since we already use that word)

Reply:
The API would then be

define:

Register
SetPreferredMimeTypeFor

query:

GetPreferredMimeTypeFor
GetPreferredFormatterFor
GetRegisteredFormatters
GetRegisteredMimeTypesFor

Reply:
That looks clearer.